packages/xo-web/src/xo-app/backup-ng/restore/index.js

```diff
@@ -161,9 +167,13 @@ export default class Restore extends Component {
           first = backup
         }
         count[backup.mode] = (count[backup.mode] || 0) + 1
+
+        if (backup.size !== undefined) {
+          size += backup.size
+        }
       })
 
-      assign(data, { first, last, count, id: vmId })
+      assign(data, { first, last, count, id: vmId, size })
```

Comment:
if a backup don't have the size property, it don't means its size is 0 byte.

Reply:
Please take care of this [comment](https://github.com/vatesfr/xen-orchestra/pull/4245#pullrequestreview-242674627) .